analyses/npl/09_water_level_events.md

```diff
@@ -207,26 +165,17 @@ for station in STATIONS:
             print(f"{station}")
             print(f"True events: {len(df_true_events)}")
```

Comment:
Think this should be changed to ```wl_events```? 

Tried running this notebook and got ```NameError: name 'df_true_events' is not defined```

Reply:
Ah oops you're right, sorry about that! I think I was running this without restarting the kernel and it was using a cached `df_true_events`.